enterprise/admin/forms.py

```diff
@@ -597,11 +598,11 @@ def clean(self):
         report_customer = cleaned_data.get('enterprise_customer')
         data_type = cleaned_data.get('data_type')
 
-        if data_type in EnterpriseCustomerReportingConfiguration.PEARSON_ONLY_REPORTS \
-                and report_customer.name != 'Pearson':
+        if data_type in EnterpriseCustomerReportingConfiguration.MANUAL_REPORTS \
```

Comment:
"MANUAL" is not a very descriptive name for these extra reports for specific enterprise customers. EXTRA_REPORTS or IN_DEPTH_REPORTS could be better options

Reply:
These are called `MANUAL_REPORTS` because everytime we need to enable these reports for a customer, we need to do lot of changes at multiple places.